drivers/overlay/ov_network.go

```diff
@@ -770,12 +770,21 @@ func (n *network) watchMiss(nlSock *nl.NetlinkSocket) {
 				}
 				n.driver.peerAdd(n.id, "dummy", ip, IPmask, mac, vtep, l2Miss, l3Miss, false)
 			} else {
-				// If the gc_thresh values are lower kernel might knock off the neighor entries.
-				// This case can happen only for local entries, from the documentation (http://man7.org/linux/man-pages/man7/arp.7.html):
+				// All the local peers will trigger a miss notification but this one is expected and the local container will reply
+				// autonomously to the ARP request
+				// In case the gc_thresh3 values is low kernel might reject new entries during peerAdd. This will trigger the following
+				// extra logs that will inform of the possible issue.
+				// Entries created would not be deleted see documentation http://man7.org/linux/man-pages/man7/arp.7.html:
 				// Entries which are marked as permanent are never deleted by the garbage-collector.
-				if time.Since(t) > 500*time.Millisecond {
+				// The time limit here is to guarantee that the dbSearch is not
+				// done too frequently causing a stall of the peerDB operations.
+				if l3Miss && time.Since(t) > 500*time.Millisecond {
```

Comment:
The `l3Miss` check was removed in an earlier commit, and already had a description for the 500ms; perhaps restore that check and description to keep the diff smaller